lib/vector2d.dart

```diff
@@ -2,74 +2,77 @@ import 'dart:math' as math;
 import 'dart:ui' as ui;
 import 'package:box2d_flame/box2d.dart' as b2d;
 
-/// An ordered pair representation (point, position, offset).
+/// An ordered pair representation (point, position, offset, vector2).
 ///
 /// It differs from the default implementations provided (math.Point and ui.Offset) as it's mutable.
 /// Also, it offers helpful converters and a some useful methods for manipulation.
 /// It always uses double values to store the coordinates.
-class Position {
+class Vector2d {
   /// Coordinates
   double x, y;
 
   /// Basic constructor
-  Position(this.x, this.y);
+  Vector2d(this.x, this.y);
 
-  /// Creates a point at the origin
-  Position.empty() : this(0.0, 0.0);
+  /// Creates a [Vector2d] with 0 splat into both lanes of the vector
```

Comment:
I think coordinates is a bit clearer than lanes. But overall I think at the origin is even clearer...